mailpoet/lib/Services/AuthorizedSenderDomainController.php

```diff
@@ -311,4 +311,25 @@ public function isAuthorizedDomainRequiredForNewCampaigns(): bool {
   public function isAuthorizedDomainRequiredForExistingCampaigns(): bool {
     return $this->restrictionsApply() && $this->isBigSender();
   }
+
+  public function getContextData(): array {
+    return [
+      'verifiedSenderDomains' => $this->getFullyVerifiedSenderDomains(true),
+      'partiallyVerifiedSenderDomains' => $this->getPartiallyVerifiedSenderDomains(true),
+      'allSenderDomains' => $this->getAllSenderDomains(),
+      'senderRestrictions' => [
+        'lowerLimit' => self::LOWER_LIMIT,
+        'upperLimit' => self::UPPER_LIMIT,
```

Comment:
`upperLimit` seems not to be part of the TSX definition and not used anywhere. I wonder what this value is good for.

Reply:
It is a leftover from a first iteration. I have added it to the cleanup ticket.